docs/ru/advancedClover.md

```diff
@@ -0,0 +1,80 @@
+# Advanced Clover
+
+Как-то раз устав выполнять некоторые действия с кловером мы решили их упростить, и так появился проект advacnedClover, которой включает в себя следующие элементы:
+
+* ide
+  * VSCode-like ide в браузере
+  * Интегрированный файловый менеджер
+  * Интегрированный терминал
+  * Автокомплит
+  * Возможность просмотра топиков
+  * Возможность просмотра топиков с изображениями
+  * Возможность вызова сервисов
+  * Возможность просмотра статуса коптера
+* Библиотека для Python
+  * Упрощение работы с навигацией
+  * Упрощение работы с камерой
+
+![Пример автокомплита, терминала и файлового менеджера ](https://i.imgur.com/9JwgXQH.png)
```

Comment:
Embedding images from external resources is not allowed. Please commit images to the repo (compress with online optimizers, use smaller resolution or jpg if they are too large).

Reply:
Fixed